package.json

```diff
@@ -18,6 +18,7 @@
   },
   "devDependencies": {
     "chai-dom": "^1.4.3",
+    "file-loader": "^0.11.2",
```

Comment:
I think you meant to add these modules to the project template package.json, this is the maji module package.json.